public/home.html

```diff
@@ -16,5 +16,10 @@ <h2>H o m e P a g e</h2>
     <p>
         <a href="home2.html" target="_self">next page</a>
         </p>
+        <form action="/somewhere_else" method="POST">…</form>
+
+        <p>
+            <a href="/somewhere_else" target="_self">maybe static page</a>
```

Comment:
`/somewhere_else` should be `/create.html`, etc., according to where the particular link needs to go next

Reply:
`target` is unused and can be removed.

Reply:
ok